unattended_installer/install_functions/checks.sh

```diff
@@ -166,6 +166,27 @@ function checks_arguments() {
 
 }
 
+# Checks if the --retry-connrefused is available in curl
+function check_curlVersion() {
+
+    if [ "${sys_type}" == "yum" ]; then
+        if ! yum list installed 2>/dev/null | grep -q curl; then
```

Comment:
Couldn't this be done easier using the curl command? `if ! curl -V`

Reply:
Done. I added `> /dev/null 2>&1` to the proposed command to hide the output of the command.

Done in https://github.com/wazuh/wazuh-packages/pull/2063/commits/55b08088e52d97c87b15a608dadf48cf9b9cd70e.